index.html

```diff
@@ -3922,44 +3923,43 @@ <h3>Usage Patterns</h3>
 
       <ul>
         <li>
-The same <a>credential</a> is presented to the same <a>verifier</a> more than
-once. In this case, the <a>verifier</a> could infer that the <a>holder</a> is
-the same individual.
+The same <a>verifiable credential</a> is presented to the same <a>verifier</a>
+more than once. The <a>verifier</a> could infer that the <a>holder</a> is the
+same individual.
         </li>
         <li>
-The same <a>credential</a> is presented to different <a>verifiers</a>, and
-either those <a>verifiers</a> collude or a third party has access to
-transaction records from both <a>verifiers</a>. In this case, the observant
-party could infer that the individual presenting the <a>credential</a> is the
-same person at both services. That is, the accounts are controlled by the same
-person.
+The same <a>verifiable credential</a> is presented to different
+<a>verifiers</a>, and either those <a>verifiers</a> collude, or a third party
+has access to transaction records from both <a>verifiers</a>. An observant
+party could infer that the individual presenting the
+<a>verifiable credential</a> is the same person at both services. That is, the
+accounts are controlled by the same person.
         </li>
         <li>
-A <a>subject</a> identifier of a <a>credential</a> refers to the same
+A <a>subject</a> identifier of a <a>verifiable credential</a> refers to the same
 <a>subject</a> across multiple <a>presentations</a> or <a>verifiers</a>. Even
```

Comment:
If you're going to change this section to use "verifiable credentials", then all "presentations" need to change to be "verifiable presentation" as well.

Reply:
changed back to "credential"